lib/smart_proxy_dynflow/io_buffer.rb

```diff
@@ -71,7 +71,7 @@ def close
       def read_available!
         data = ''
         loop { data += @io.read_nonblock(4096) }
-      rescue IO::WaitReadable # rubocop:disable Lint/HandleExceptions
```

Comment:
This was changed to `Lint/SuppressedException`, which is why it now shows up in the TODO file